### PR TITLE
orcidutils flake8 compliance

### PR DESCRIPTION
**Pull request by tsgit**
I was looking at orcidutils.py because of https://app.asana.com/0/3728178550980/25419738098924
and fixed 2 minor issues along with some major cleanup for pylint/flake8. Many unused imports removed.

1) "if x == None"  rewritten to "if x is None"
2) missing space inserted in error message for duplicate arxiv ids

Note that import-order was decided by RFC to adhere to pep8 and should be checked with e.g. flake8. So I fixed that here, too.

### CODE REVIEW

**Comment by kaplun** [comment]
Hi Thorsten,

Can you just rebase your branch, as it does not merge anymore... (I am sorry I have seen it after other things been merged)...

File: modules/miscutil/lib/orcidutils.py [hunk 1]
```diff
@@ -675,8 +661,8 @@ def _stub_method():
                     _stub_method()
                 except DoubledIds:
                     register_exception(subject="The paper %s contains the"
-                                       "same ArXiv id %s twice." % (recid,
-                                                                    encoded),
+                                       " same arXiv id %s twice."
```
Comment (tsgit) [inline modules/miscutil/lib/orcidutils.py:664]: this was the original reason for editing -- inserting a missing space in the exception message

File: modules/miscutil/lib/orcidutils.py [hunk 2]
```diff
@@ -19,45 +19,31 @@
...
-from invenio.bibauthorid_dbinterface import get_name_by_bibref
-from invenio.bibauthorid_dbinterface import get_orcid_id_of_author
-from invenio.bibauthorid_dbinterface import get_papers_of_author
-from invenio.bibauthorid_dbinterface import \
-    get_personid_signature_association_for_paper
-from invenio.bibauthorid_dbinterface import get_signatures_of_paper
-from invenio.bibauthorid_dbinterface import get_token
-from invenio.bibauthorid_dbinterface import trigger_aidtoken_change
+from invenio.bibauthorid_dbinterface import (delete_token, get_all_tokens,
+                                             get_doi_for_paper,
```
Comment (tsgit) [inline modules/miscutil/lib/orcidutils.py:31]: import order and aggregation via  isort

**Comment by tsgit** [comment]
sorry this has languished for a while. I rebased and adopted isort for import order. these are very minor cosmetic changes. a bunch of unused imports were removed.

pylint, kwalitee, and flake8 are happier, although there is a name conflict between

import orcid 
and function variables in 2 places
W:370,64: Redefining name 'orcid' from outer scope (line 24) (redefined-outer-name)
W:586,63: Redefining name 'orcid' from outer scope (line 24) (redefined-outer-name)

the only real change is in the exception message line 664, where a missing space is inserted.
T.

**Comment by jalavik** [comment]
@tsgit Thanks. @kaplun Shall we merge and deploy?

**Comment by kaplun** [comment]
Ah sure :-) Why not?

**Pull request merged**